lib/spike-detector.js

```diff
@@ -0,0 +1,40 @@
+var moment = require('moment');
+
+var normalTraffic = {};
```

Comment:
this should be scoped to `new SpikeD(config);` object. 

so below you should reference `this.normalTraffic`